### PR TITLE
chore: add optimism sepolia deployment config

### PR DESCRIPTION
**Pull request by 0x-r4bbit**

### CODE REVIEW

**Review by osmaczko — APPROVED** [review]

**Comment by 0x-r4bbit** [comment]
@felicio merging is still blocked here due to missing review.

Do you think we can get this in? Notice though that this only updates the contract deployment config and not the DApp's config.

File: packages/contracts/script/DeploymentConfig.s.sol [hunk 1]
```diff
@@ -40,11 +40,15 @@ contract DeploymentConfig is Script {
...
     address internal SNT_ADDRESS_OPTIMISM_GOERLI = 0xcAD273fA2bb77875333439FDf4417D995159c3E1;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal SNT_ADDRESS_OPTIMISM_SEPOLIA = 0x0B5DAd18B8791ddb24252B433ec4f21f9e6e5Ed0;
 
     // solhint-disable-next-line var-name-mixedcase
     address internal MULTICALL_ADDRESS_OPTIMISM = 0xeAa6877139d436Dc6d1f75F3aF15B74662617B2C;
     // solhint-disable-next-line var-name-mixedcase
     address internal MULTICALL_ADDRESS_OPTIMISM_GOERLI = 0xcA11bde05977b3631167028862bE2a173976CA11;
+    // solhint-disable-next-line var-name-mixedcase
+    address internal MULTICALL_ADDRESS_OPTIMISM_SEPOLIA = 0xcA11bde05977b3631167028862bE2a173976CA11;
```
Comment (felicio) [inline packages/contracts/script/DeploymentConfig.s.sol:51]: non-blocking: do you know why `MULTICALL_ADDRESS_OPTIMISM_SEPOLIA` and `MULTICALL_ADDRESS_OPTIMISM_GOERLI` have the same address?

File: packages/contracts/script/DeploymentConfig.s.sol
Comment (felicio) [inline packages/contracts/script/DeploymentConfig.s.sol:1]: non-blocking: could you update `packages/contracts/README.md` as well?

**Review by felicio — APPROVED** [review]
pre-approving

File: packages/contracts/script/DeploymentConfig.s.sol
Comment (felicio) [inline packages/contracts/script/DeploymentConfig.s.sol:1]: non-blocking: do you plan to verify the contracts, so they can be read too https://sepolia-optimism.etherscan.io/address/0x7Ff554af5b6624db2135E4364F416d1D397f43e6#code ?

Comment (0x-r4bbit) [reply packages/contracts/script/DeploymentConfig.s.sol:1]: Will do!

Comment (0x-r4bbit) [reply packages/contracts/script/DeploymentConfig.s.sol:1]: Ah yes, will take care of this.

**Comment by 0x-r4bbit** [comment]
@felicio updated the PR accordingly.

Thank you for your time and input!

**Comment by felicio** [comment]
@0x-r4bbit thank you, 🚀

**Pull request merged**